model/things/src/main/java/org/eclipse/ditto/model/things/ImmutableThingFromCopyBuilder.java

```diff
@@ -690,6 +694,17 @@ public FromCopy setModified(final Predicate<Instant> existingModifiedPredicate,
         return this;
     }
 
+    @Override
+    public FromCopy setMetadata(JsonPointer resourcePath, Metadata metadata) {
+        fromScratchBuilder.setMetadata(resourcePath, metadata);
+        return this;
+    }
+
+    public FromCopy setMetadata(Metadata metadata) {
```

Comment:
What is this method for - not overriding an interface method?

Reply:
@thjaeckle I was unsure about this one. This method is a shorthand and is called in the `of(JsonObject)` method as FP. I dont know if we shuld then move it up to the interface. It could even be private here, in theory or we just call the other method `(JsonPointer, Metadata)` and pass empty Pointer explicitly. What would you prefer?

Reply:
I would prefer passing the empty Pointer explicitly to the already existing interface method.

Reply:
On Thing level the Pointer is unused and  thus it should be removed IMHO.